pkg/kubelet/kubelet.go

```diff
@@ -1832,6 +1832,9 @@ func (kl *Kubelet) syncTerminatedPod(ctx context.Context, pod *v1.Pod, podStatus
 
 	// generate the final status of the pod
 	// TODO: should we simply fold this into TerminatePod? that would give a single pod update
+	if nil == podStatus {
```

Comment:
please reverse this (`podStatus == nil`)

Reply:
Hold on.  Running pods shouldn't be terminated.  So it's a bug to get into this path.

/hold

Reply:
Yeah I wasn't sure about this - (trying to figure out how it could happen rn)